lib/matplotlib/axes/_axes.py

```diff
@@ -2148,7 +2151,9 @@ def barh(self, y, width, height=0.8, left=None, *, align="center",
 
             - scalar: symmetric +/- values for all bars
             - shape(N,): symmetric +/- values for each bar
-            - shape(2,N): separate + and - values for each bar
+            - shape(2,N): Separate - and + values for each bar. First row
```

Comment:
I don't think this sentence will be very clear to new users. Could you add an Example numpydoc section with a short example example demonstrating the clarification?

Reply:
I added a link to [this example](https://matplotlib.org/gallery/statistics/errorbar_features.html?highlight=errorbar%20sphinx%20gallery).